actors/test/power_scenario_test.go

```diff
@@ -110,12 +110,16 @@ func TestOnEpochTickEnd(t *testing.T) {
 		// Original send to storage power actor
 		To:     builtin.StoragePowerActorAddr,
 		Method: builtin.MethodsPower.OnEpochTickEnd,
-		SubInvocations: []vm.ExpectInvocation{{
-			// expect call to reward to update kpi
-			To:     builtin.RewardActorAddr,
-			Method: builtin.MethodsReward.UpdateNetworkKPI,
-			From:   builtin.StoragePowerActorAddr,
-		}},
+		SubInvocations: []vm.ExpectInvocation{
+			{To: builtin.RewardActorAddr, Method: builtin.MethodsReward.ThisEpochReward},
```

Comment:
nit: since this test is very commenty it would be nice to summarize these two lines in a comment too